explorer/client/src/pages/object-result/ObjectLoaded.tsx

```diff
@@ -88,99 +79,23 @@ function ObjectLoaded({ data }: { data: DataType }) {
-        if (addrOwner.length !== 20) {
-            console.log('address owner byte length must be 20');
-            return null;
-        }
-
-        return asciiFromNumberBytes(addrOwner);
-    };
-    type SuiIdBytes = { bytes: number[] };
-
-    function handleSpecialDemoNameArrays(data: {
-        name?: string;
-        player_name?: SuiIdBytes | string;
-        monster_name?: SuiIdBytes | string;
-        farm_name?: SuiIdBytes | string;
-    }): string {
-        let bytesObj: SuiIdBytes = { bytes: [] };
-
-        if ('player_name' in data) {
-            bytesObj = data.player_name as SuiIdBytes;
-            const ascii = asciiFromNumberBytes(bytesObj.bytes);
-            delete data.player_name;
-            return ascii;
-        } else if ('monster_name' in data) {
-            bytesObj = data.monster_name as SuiIdBytes;
-            const ascii = asciiFromNumberBytes(bytesObj.bytes);
-            delete data.monster_name;
-            return ascii;
-        } else if ('farm_name' in data) {
-            bytesObj = data.farm_name as SuiIdBytes;
-            const ascii = asciiFromNumberBytes(bytesObj.bytes);
-            delete data.farm_name;
-            return ascii;
-        } else if ('name' in data) {
-            return data['name'] as string;
-        } else {
-            bytesObj = { bytes: [] };
-        }
-
-        return asciiFromNumberBytes(bytesObj.bytes);
-    }
-
-    function toHexString(byteArray: number[]): string {
-        return (
-            '0x' +
-            Array.prototype.map
-                .call(byteArray, (byte) => {
-                    return ('0' + (byte & 0xff).toString(16)).slice(-2);
-                })
-                .join('')
-        );
-    }
-
-    function processName(name: string | undefined) {
-        // hardcode a friendly name for gas for now
-        const gasTokenTypeStr = 'Coin::Coin<0x2::GAS::GAS>';
-        const gasTokenId = '0000000000000000000000000000000000000003';
-        if (data.objType === gasTokenTypeStr && data.id === gasTokenId)
-            return 'GAS';
-
-        if (!name) {
-            return handleSpecialDemoNameArrays(data.data.contents);
-        }
-    }
-
-    function processOwner(owner: any) {
-        if (typeof owner === 'object' && 'AddressOwner' in owner) {
-            return toHexString(owner.AddressOwner);
-        }
-
-        return owner;
-    }
-
     const viewedData = {
         ...data,
         objType: trimStdLibPrefix(data.objType),
-        name: processName(data.name),
-        tx_digest:
-            data.data.tx_digest && typeof data.data.tx_digest === 'object'
-                ? toHexString(data.data.tx_digest as number[])
-                : data.data.tx_digest,
-        owner: processOwner(data.owner),
+        name: data.name,
+        tx_digest: data.data.tx_digest,
+        owner: data.owner,
         url: parseImageURL(data.data.contents),
     };
 
     //TO DO remove when have distinct name field under Description
```

Comment:
Do you know what does this function mean?

Reply:
it's left over, i'll remove. it just meant that we were using a regex for the name